README.md

```diff
@@ -258,6 +258,20 @@ Congratulations! You have successfully started the flow. Carry on reading the ne
   }
   ```
 
+- **`onUserExit {Function} optional`**
+
+  Callback that fires when the user abandons the flow without completing it. The callback returns a string with the reason for leaving. When the user exits the flow by declining the consent prompt the value returned will be `'USER_CONSENT_DENIED'`.
+
+  ```js
+  Onfido.init({
+    token: 'your-jwt-token',
+    containerId: 'onfido-mount',
+    onUserExit: function (userExitCode) {
+      console.log(userExitCode)
+    },
+  })
+  ```
+
```

Comment:
I can't move this from here because this is a new callback that needs to be documented in the Handling Callback sections. Having said that, after your comment I spoke to See Wah and we decided to add this note in the userConsent step section

> **Note**: The `userConsent` step must be used in conjunction with the `onUserExit` callback. See the [Handling Callbacks](#handling-callbacks) section for more information.